Content.Shared/Mind/SharedMindSystem.cs

```diff
@@ -652,6 +652,23 @@ public HashSet<Entity<MindComponent>> GetAliveHumans(EntityUid? exclude = null,
 
         return allHumans;
     }
+
+    // Begin Delta-V Changes
+
+    /// <summary>
+    /// Set whether or not to show examine information about the mind. Used to obscure if a mind is SSD or not.
+    /// </summary>
+    /// <param name="uid">Entity to set the mind examine info for.</param>
+    /// <param name="showExamineInfo">True to show examine information, false to hide it.</param>
+    public void ShowExamineInfo(EntityUid uid, bool showExamineInfo)
+    {
+        if (!TryComp<MindContainerComponent>(uid, out var mindContainer))
+            return;
+
+        mindContainer.ShowExamineInfo = showExamineInfo;
+        Dirty(uid, mindContainer);
+    }
+    // End Delta-V Changes
```

Comment:
move this to a partial system in _DV

Reply:
Didn't notice the base class was partial, easy fix.

Reply:
even if it wasnt you can just make it partial